index.js

```diff
@@ -2441,6 +2441,30 @@
         return n < 0 || negativeZero(n) ? Nothing() : slice(0, -n, xs);
       });
 
+  //# reverse :: [a] -> [a]
+  //.
+  //. Returns the elements of the given list in reverse order.
+  //.
+  //. ```javascript
+  //. > S.reverse([1, 2, 3])
+  //. [3, 2, 1]
+  //.
+  //. > S.reverse('abc')
+  //. 'cba'
+  //. ```
+  S.reverse =
+  def('reverse',
+      {},
+      [List(a), List(a)],
+      function reverse(xs) {
+        var result = [];
+        if (_type(xs) === 'String') return reverse(xs.split('')).join('');
```

Comment:
No problem - I just usually put it on top since it does not matter anyhow because of variable hoisting. I'll move it.